models/donor.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require("mongoose");
+const donorSchema = new mongoose.Schema({
+  donorName: String,
```

Comment:
the schema is for donor so no need to prefix donor in it. Change it to "name", "bloodgorup", etc